scripts/memory-threshold-list.yaml

```diff
@@ -1,41 +1,83 @@
 - scenarios:
     - sample.matter.lock.release
     - sample.matter.light_bulb.release
+    - sample.matter.window_covering.release
+    - sample.matter.light_switch.release
   platforms:
     - nrf52840dk_nrf52840
-  ram_size: 241664
-  rom_size: 778240
+  ram_size: 183500 # 30% of free space
+  rom_size: 786022 # 20% of free space
   codeowners:
     - kkasperczyk-no
     - ArekBalysNordic
 
 - scenarios:
-    - sample.matter.lock.debug
-    - sample.matter.light_bulb.debug
+    - sample.matter.template.release
   platforms:
     - nrf52840dk_nrf52840
-  ram_size: 241664
-  rom_size: 931840
+  ram_size: 183500 # 30% of free space
```

Comment:
For now, it just notifies code owners, but we're wondering, and considering blocking PRs that overlap those values - by requesting changes or something different and it will require an action from the code owners - approving + increase limits or something else. Regarding the second thing, the small warning sounds reasonable :)